lab_1/main.py

```diff
@@ -62,10 +98,20 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
     :param top_n: a number of the most common words
     :return: the distance
     """
-    pass
+    if isinstance(unknown_profile, dict)\
+            and isinstance(profile_to_compare, dict) and isinstance(top_n, int):
+        cross = []
+        for i in get_top_n_words(profile_to_compare['freq'], top_n):
+            if i in get_top_n_words(unknown_profile['freq'], top_n):
+                cross.append(i)
+        cross = float(len(cross) / top_n)
+        cross = round(cross, 2)
```

Comment:
you can join this line and the previous one

Reply:
Done